### PR TITLE
External UCS Consent_ON: Merge scripts from custom project

### PR DESCRIPTION
**Pull request by dboltovskyi**
Scripts are part of test set: https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/1301
Note: scripts 011 - 013 will be provided in another PR

### CODE REVIEW

**Comment by dboltovskyi** [comment]
Common functions for CRQ: https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/1211

**Comment by dboltovskyi** [comment]
Postponed due to updates are required

File: test_scripts/Polices/External_UCS/External_Consent_ON/ATF_Policies_External_Consent_ON_OFF_combine.lua [hunk 1]
```diff
@@ -0,0 +1,216 @@
+------------------------------------------------------------------------------------------------------
```
Comment (dboltovskyi) [inline test_scripts/Polices/External_UCS/External_Consent_ON/ATF_Policies_External_Consent_ON_OFF_combine.lua]: @istoimenova, Accepted

Comment (dboltovskyi) [inline test_scripts/Polices/External_UCS/External_Consent_ON/ATF_Policies_External_Consent_ON_OFF_combine.lua]: @istoimenova, Accepted
Test set is in https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/1301

Comment (dboltovskyi) [inline test_scripts/Polices/External_UCS/External_Consent_ON/ATF_Policies_External_Consent_ON_OFF_combine.lua]: @istoimenova, Accepted

File: test_scripts/Polices/External_UCS/External_Consent_ON/ATF_Policies_External_Consent_ON_OFF_combine.lua [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+------------------------------------General Settings for Configuration--------------------------------
```
Comment (dboltovskyi) [inline test_scripts/Polices/External_UCS/External_Consent_ON/ATF_Policies_External_Consent_ON_OFF_combine.lua]: @istoimenova, Rejected.
Please clarify which errors should be fixed?

File: test_scripts/Polices/External_UCS/External_Consent_ON/001_ATF_Policies_External_Consent_ON_disallowed_by_external_consent_entities_off_data_consent_disallowed.lua [hunk 3]
```diff
@@ -0,0 +1,150 @@
+-------------------------------------- Requirement summary -------------------------------------------
```
Comment (istoimenova) [inline test_scripts/Polices/External_UCS/External_Consent_ON/001_ATF_Policies_External_Consent_ON_disallowed_by_external_consent_entities_off_data_consent_disallowed.lua]: @dboltovskyi, as I understand scripts 011,012,013 should be newly prepared.

File: test_scripts/Polices/External_UCS/External_Consent_ON/001_ATF_Policies_External_Consent_ON_disallowed_by_external_consent_entities_off_data_consent_disallowed.lua [hunk 4]
```diff
@@ -1,3 +1,6 @@
...
```
Comment (dboltovskyi) [inline test_scripts/Polices/External_UCS/External_Consent_ON/001_ATF_Policies_External_Consent_ON_disallowed_by_external_consent_entities_off_data_consent_disallowed.lua]: @istoimenova, Please notice, there was made an agreement with management not to fix such issues right now and to move this task to backlog.
Appropriate task was created, with "Fix LuaCheck issues in scripts" summary

File: test_scripts/Polices/External_UCS/External_Consent_ON/001_ATF_Policies_External_Consent_ON_disallowed_by_external_consent_entities_off_data_consent_disallowed.lua (hunk 3 above)
Comment (dboltovskyi) [inline test_scripts/Polices/External_UCS/External_Consent_ON/001_ATF_Policies_External_Consent_ON_disallowed_by_external_consent_entities_off_data_consent_disallowed.lua]: @istoimenova, Correct

**Pull request merged**